data/boarding missions.txt

```diff
@@ -24,6 +24,356 @@ mission "Assisting Merchant"
+		payment 10000
+		conversation
+			`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment> along with a message saying, "May fair winds and bright skies go with you!"`
+				decline
+	destination Earth
+
+mission "Assisting Merchant (Force)"
+	assisting
+	repeat
+	to offer
+		random < 5
+	source
+		government Merchant
+	on offer
+		payment 10000
+		conversation
+			`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment> along with a message saying, "May the Force be with you!"`
+				decline
+	destination Earth
+	
+mission "Assisting Merchant (Navy)"
+	assisting
+	repeat
+	to offer
+		random < 10
+	source
+		government Merchant
+	on offer
+		payment 10000
+		conversation
+			`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment> along with a message saying, "The best way to be safe from pirates is to stick to systems patrolled by the Navy."`
+				decline
+	destination Earth	
+	
+mission "Assisting Pirates (Bastard)"
+	assisting
+	repeat
+	to offer
+		random < 12
+	source
+		government Pirate
+	on offer
+		payment -100000
+		conversation
+			`After assisting the <origin> you find that 100000 credits have been removed from your account by those blasted pirates.`
+				decline
+	destination Greenrock
+
+mission "Assisting Pirates (Setback)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government Pirate
+	on offer
+		payment 100
+		conversation
+			`The captain of the <origin> gives you <payment> for "helping him out of a temporary setback."`
+				decline
+	destination Greenrock
+	
+mission "Assisting Pirates (Threat)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government Pirate
+	on offer
+		payment 100
+		conversation
+			`The captain of the <origin> gives you <payment> and a message saying, "You better watch who you mention this to, I'll be watching."`
+				decline
+	destination Greenrock
+
+mission "Assisting Pirates (Weed)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government Pirate
+	on offer
+		payment 100
+		conversation
+			`The captain of the <origin> gives you <payment> along with lucrative offer for space weed. "Only 6000 Credits a Pound!"`
+				decline
+	destination Greenrock
+	
+mission "Assisting Syndicate"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment>.`
+				decline
+	destination Foundry
+
+mission "Assisting Syndicate (Protector)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment> along with a message reading, "Trouble with Pirates? Hire a Syndicate Protector and your troubles will be over."`
```

Comment:
Seeing as you can't hire any ships this is a bit odd. Buying a protector would be fine though.

Reply:
once again, the words from a hail not mine, but i see your point